gatling/gatling.py

```diff
@@ -171,14 +171,22 @@ def sink(selector, not_done):
                 if not p:
                     if verbose:
                         running = len(p_filled) + len(p_open)
-                        print(f"# {my_name} STARTED A PROCESS (1 + {running} + {len(res)}):",
-                              *cmd,
-                              file=sys.stderr)
-                    p = Popen(cmd, encoding=stdout.encoding, stdin=PIPE, stdout=PIPE, stderr=PIPE)
+                        print(
+                            f"# {my_name} STARTED A PROCESS (1 + {running} + {len(res)}):",
+                            *cmd,
+                            file=sys.stderr,
+                        )
+                    p = Popen(
+                        cmd,
+                        encoding=stdout.encoding,
+                        stdin=PIPE,
+                        stdout=PIPE,
+                        stderr=PIPE,
+                    )
```

Comment:
Urgh, this just makes me sad.

Reply:
Eh, I'm not mad at it. Some projects that I've worked on in the past observe the 80 char width limit and this is the approach that I ended up adopting.